airflow/providers/facebook/ads/hooks/ads.py

```diff
@@ -55,9 +55,15 @@ class FacebookAdsReportingHook(BaseHook):
 
     """
 
+    conn_name_attr = 'facebook_conn_id'
+    default_conn_name = 'facebook_default'
+    conn_type = 'facebook_social'
+    hook_name = 'Facebook Social'
```

Comment:
Should we call it "Facebook Ads" instead

Reply:
Good question. I took it from "extra-packages" reference. But when you look at the API it is actually called 'Facebook Marketing API" - but indeed this one uses the "Ads" part. I will replace it here but leave `Facebook Social` in "extras" as there might be more operators added in the future - both marketing and non-marketing ones.

Reply:
'facebook_social` would remain as connection name thought. I do not want to break compatibility